[US] Add configurable camera speed

kkykkykky opened the pull request:
A long time ago Joan accepted this small change I proposed, but, unfortunately, he never got around to merging it into his code so it was never released.

As you know, things in AI and HS2 are 10 times bigger, so the camera movement speed boost that UsefulStuff offers is not very sufficient. This change allows users to customize the speed to their liking. I've set the default factor for AI and HS2 to 10.

I've tested it in AI, HS2, and KKS. The KK I have is too old, so I'm not able to test.

Thanks for forking HSPlugins!

ManlyMarco commented on UsefulStuff.Core/Features/CameraShortcuts.cs:
```diff
@@ -60,9 +60,9 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
             {
                 if (Input.GetKey(KeyCode.LeftControl))
-                    return Input.GetAxis(axis) / 6f;
+                    return Input.GetAxis(axis) / (6f * HSUS.CameraMultiplierFactor.Value);
```
Shouldn't this divide after multiplying by the correction factor? The way it is now the camera will move even slower in AI/HS2 when everything has a larger scale, so it'll act differently from the KK line of games.

kkykkykky replied:
If I just did that, the slow down function would become a speed up function. But I do agree that the camera moves even slower with the larger scale which isn't the expected behavior. Thanks for catching this. This is why you're Manly and I'm just KKY 😁. I guess the underlying issue is that the vanilla camera movement speed in AI/HS2 is 10 times too slow to begin with.

Would this be ok?
```csharp
            public static float GetCameraMultiplier()
            {
                float _adjDeltaTime = Time.deltaTime * HSUS.CameraMultiplierFactor.Value;
                if (Input.GetKey(KeyCode.LeftControl))
                    return _adjDeltaTime / 6f;
                if (Input.GetKey(KeyCode.LeftShift))
                    return _adjDeltaTime * 4f;
                return _adjDeltaTime;
            }
```
Maybe I should make the `6f` and `4f` configurable as well. What do you think?

ManlyMarco replied:
I think it's fine.

You could have "FastCamSpeed" and "SlowCamSpeed" settings that replace the constant values (instead of dividing by 6f it should multiply by 0.6f in that case so that both settings have values in the same units). In this case you'd want to replace `Time.deltaTime * HSUS.CameraMultiplierFactor.Value;` with a hardcoded `* 10f` for AI/HS2 to not have to have different defaults for the settings.

kkykkykky commented:
Ok. Added configurable CamSpeedFast and CamSpeedSlow. Removed the multiplier from mouse control as it doesn't seem beneficial during my test. Changed when CameraShortcuts value is checked. Also noticed a few config keys were mismatched. Not sure if it's intentional or not. Some of them don't seem to actually do anything at the moment, but I changed them anyway.
